### ISSUE TITLE
Screenshot in readme missing alt text

### ISSUE THREAD

**Issue by tspivey** [kennethreitz/records#53]
The screenshot of the usage information is missing its alt attribute.
As a result, my screen reader will read its URL, which isn't very useful.

**Comment by kennethreitz** [comment]
Ah, thank you! I'll fix that now.

**Comment by kennethreitz** [comment]
This is now fixed. :)

:sparkles: :cake: :sparkles:
